benchmark/src/worker.ts

```diff
@@ -18,13 +22,29 @@ async function main() {
   // use in-memory storage without filesystem persistence
   app.bind('datasources.config.db').to({connector: 'memory'});
 
+  // overwrite the error logger to print all failed requests, including 4xx
+  app.bind(RestBindings.SequenceActions.LOG_ERROR).to(logAllErrors);
+
   await app.start();
 
   // Tell the master thread what is our URL
-  console.log(app.restServer.url);
+  console.log('Server listening at', app.restServer.url);
```

Comment:
nitpick: add a space after `at`?

Reply:
Please note `console.log` is already adding one space character to delimit arguments.

Reply:
```
$ node
Welcome to Node.js v12.13.1.
Type ".help" for more information.
> console.log('Server listening at', 'http://localhost')
Server listening at http://localhost
undefined
```